.azure/azure-linux-template.yml

```diff
@@ -8,7 +8,7 @@ jobs:
         BUILD: stack
       stack-lts-11:
         BUILD: stack
-        ARGS: "--resolver lts-11"
+        ARGS: "--stack-yaml stack.lts-11.yaml"
```

Comment:
It's still a good idea, though not required, to keep the `--resolver lts-11` in addition to the `--stack-yaml`, since `--resolver lts-11` will force the latest lts-11 minor release to be used. Since there are unlikely to be any new lts-11 releases, I'm not too bothered by this.